src/app/interfaces/filter.ts

```diff
@@ -0,0 +1,27 @@
+export type valueProduct = string | number | ValueObjectParameters;
+
+export interface OptionsObjectFilter {
+  type: string;
+  value: valueProduct;
+}
+
+export interface SelectOptions {
+  [key: string | symbol]: Array<SelectObject>;
+}
+
+interface ValueObjectParameters {
+  minValue: number;
+  maxValue: number;
+}
+
+export interface SelectObject {
+  value:
```

Comment:
Maybe separate types which extends from basic interface? To not select string OR number OR another one?